apps/web/src/components/commands/tokens/helpers.ts

```diff
@@ -0,0 +1,17 @@
+export enum supportedChains {
```

Comment:
Using the `wagmi/chains` would be nice here!

Reply:
Haha was my previous comment!

Reply:
It would need to extend in that case! The `Portals.Fi` supported some chains that `wagmi/chains` didn't and vice versa.